vendor/Ltac2/README.md

```diff
@@ -0,0 +1,25 @@
+[![Build Status](https://travis-ci.org/ppedrot/ltac2.svg?branch=master)](https://travis-ci.org/ppedrot/ltac2)
+
+Overview
+========
+
+This is a standalone version of the Ltac2 plugin. Ltac2 is an attempt at
+providing the Coq users with a tactic language that is more robust and more
+expressive than the venerable Ltac language.
+
+Status
+========
+
+It is mostly a toy to experiment for now, and the implementation is quite
+bug-ridden. Don't mistake this for a final product!
+
+Installation
+============
+
+This should compile with Coq master, assuming the `COQBIN` variable is set
+correctly. Standard procedures for `coq_makefile`-generated plugins apply.
+
+Demo
+====
+
+Horrible test-files are provided in the `tests` folder. Not for kids.
```

Comment:
If the tests are indeed horrible it's going to make it harder for others to understand and maintain Ltac2.
Comments identifying the purpose of each test or each group of tests are invaluable.

Reply:
They are horrible as generic code examples, not as tests. The doc means "don't read them to learn programming in Ltac2".

Reply:
Hem, then may I suggest just removing this section from the README? It is like saying "don't think about the pink elephant"...

And I think nobody looks at test-suite as a set of examples ;-)

Reply:
I'll remove the `README` as I find the manual is a much better place. The test examples are no longer there anyway (they were moved to the test suite).